example/index.php

```diff
@@ -35,10 +35,11 @@
 $httpClient = new \SocialConnect\Common\Http\Client\Cache(
     $httpClient,
     /**
-     * Please dont use FilesystemCache for production/stage env, only for local testing!
-     * It doesnot support cache expire (remove)
+     * You can use any library with PSR-6 compatibility
```

Comment:
Could you use PSR-16 instead? CakPHP doesn't have PSR-6 support. Symfony cache has adapters for both so it should be a problem for it.

Reply:
Done, and I've prepare a huge number of improvements for `Http\Client\Cache` that use this cache 😸